qt/widgets/mplcpp/inc/MantidQtWidgets/MplCpp/MantidColorMap.h

```diff
@@ -19,11 +19,6 @@ namespace MantidQt {
 namespace Widgets {
 namespace MplCpp {
 
-/**
- * @brief The MantidColormap exists to provide an matplotlib-based
- * implementation that satisfies the same colormap interface used by the Qt4
- * Qwt-based version in the Plotting library.
- */
```

Comment:
Maybe leave this comment in to allow for historical knowledge for this class! Worth double checking this MantidColormap is indeed used currently in workbench, but it probably is!